package.json

```diff
@@ -134,8 +134,8 @@
     "@elastic/charts": "^30.2.0",
     "@elastic/datemath": "^5.0.3",
     "@elastic/eslint-config-kibana": "^0.15.0",
-    "@svgr/core": "5.4.0",
-    "@svgr/plugin-svgo": "^4.0.3",
+    "@svgr/core": "^6.5.1",
+    "@svgr/plugin-svgo": "^6.5.1",
```

Comment:
Is there any reason these aren't being updated to `7.0.0`?

Reply:
Version 7 came out recently, thanks, I updated to the latest